### PR TITLE
update contract addresses for zn sepolia

### PR DESCRIPTION
**Pull request by isaaczaak**
- current live version is just a full copy of the mainnet addresses
- replaced all the addresses with accurate ones

### CODE REVIEW

File: docs/zora-network/network.mdx [hunk 1]
```diff
@@ -66,23 +66,21 @@ Zora Sepolia Network is an Testnet L2 built on top of Ethereum Sepolia using the
...
-| Proxy\_\_OVM_L1CrossDomainMessenger | 0xdC40a14d9abd6F410226f1E6de71aE03441ca506 |
-| L1ERC721BridgeProxy                 | 0x83A4521A3573Ca87f3a971B169C5A0E1d34481c3 |
-| SystemConfig                        | 0x17fb7c8Ce213F1A7691ee41EA880ABf6eBC6fa95 |
-| OptimismPortalProxy                 | 0x1a0ad011913A150f69f6A19DF447A0CfD9551054 |
-| L1CrossDomainMessenger              | 0x363B4B1ADa52E50353f746999bd9E94395190d2C |
-| L2OutputOracle                      | 0x89336159Edd615260a95309e46343602D6b6489e |
-| SystemDictatorProxy                 | 0x50bA02c0Ca5E7bF772913EaF24Fb1fd3842a5f19 |
-| SystemConfigProxy                   | 0xA3cAB0126d5F504B071b81a3e8A2BBBF17930d86 |
+| OptimismMintableERC20FactoryProxy   | 0x5F3bdd57f01e88cE2F88f00685D30D6eb51A187c |
+| OptimismMintableERC20Factory        | 0x42D4a53fdcbc738479aaaC9a39e3eFe6A3216Bef |
```
Comment (IsabellaSmallcombe) [inline docs/zora-network/network.mdx:70]: @isaaczaak i'm not seeing these contracts show up on the explorer. for example https://sepolia.explorer.zora.energy/address/0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2

Comment (iainnash) [reply docs/zora-network/network.mdx:70]: this particular contract and many others are on sepolia: 
https://sepolia.etherscan.io/address/0x42D4a53fdcbc738479aaaC9a39e3eFe6A3216Bef

Comment (IsabellaSmallcombe) [reply docs/zora-network/network.mdx:70]: this is for zora network no?

**Comment by iainnash** [comment]
Some contracts are on ZN and some are on sepolia to bridge over and back.

File: docs/zora-network/network.mdx (hunk 1 above)
Comment (iainnash) [reply docs/zora-network/network.mdx:70]: The l1 part of the zora network is on sepolia.

**Review by iainnash — APPROVED** [review]

**Comment by oveddan** [comment]
we should close this repo and migrate this change to the monorepo docs

**Pull request merged**